bolt/test/X86/bb-with-two-tail-calls.s

```diff
@@ -1,8 +1,6 @@
 # This reproduces a bug with dynostats when trying to compute branch stats
 # at a block with two tails calls (one conditional and one unconditional).
 
-# REQUIRES: system-linux
```

Comment:
why has this been removed?

Reply:
To test on a mac :)
It's not needed in the first place because the test is OS-agnostic.